apps/barista-design-system/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -154,6 +173,16 @@ export class BaIconColorWheel {
     this._iconId = `ba-icon-color-wheel-iconId-${++uniqueColorwheelId}`;
   }
 
+  ngAfterViewInit(): void {
+    if (!this.isBrowser) {
```

Comment:
We can use the renderer in both cases, can't we? 
I don't see why we need to complicate this